cmd/ww/server.go

```diff
@@ -344,7 +348,20 @@ func server(args ...string) {
 		IdleTimeout:  20 * time.Second,
 		Addr:         *httpsaddr,
 		Handler:      http.HandlerFunc(handler),
-		TLSConfig:    &tls.Config{GetCertificate: m.GetCertificate},
+		// Mozilla intermediate TLS configuration
+		// https://ssl-config.mozilla.org/#server=go&config=intermediate
+		TLSConfig: &tls.Config{
+			GetCertificate: m.GetCertificate,
+			MinVersion:     tls.VersionTLS12,
```

Comment:
i think we can get away with setting this to tls1.3 and avoid having to list the cipher suites by hand.

my logic: we require wasm, which means we have defined (and somewhat restrictive) limits on how far backwards compatible we need to be. according to caniuse.com, tls1.3 is supported by every major browser that supports wasm, and only came a few versions behind it. https://caniuse.com/wasm https://caniuse.com/tls1-3

the downside is that users on browsers that don't support tls1.3 will get a browser error page, instead of a webwormhole message explaining that they need a more modern browser. i think it's worth trying anyway.

Reply:
I think you have a point so I agree. I'll change it to TLS 1.3.